source/module_elecstate/module_charge/symmetry_rho.cpp

```diff
@@ -17,96 +16,137 @@ void Symmetry_rho::begin(const int& spin_now,
                          Parallel_Grid& Pgrid,
                          ModuleSymmetry::Symmetry& symm) const
 {
-	assert(spin_now < 4);//added by zhengdy-soc
+    assert(spin_now < 4); // added by zhengdy-soc
+
+    if (ModuleSymmetry::Symmetry::symm_flag != 1) {
+        return;
+}
+    // both parallel and serial
+    // if(symm.nrot==symm.nrotk) //pure point-group, do rho_symm in real space
+    // {
+    // 	psymm(CHR.rho[spin_now], rho_basis, Pgrid, symm);
+    // 	if(XC_Functional::get_func_type() == 3 || XC_Functional::get_func_type() == 5) psymm(CHR.kin_r[spin_now],
+    // rho_basis,Pgrid,symm);
+    // }
+    // else	//space group, do rho_symm in reciprocal space
+    {
```

Comment:
what is this '{' used for?

Reply:
The extra pair of braces was originally added to enclose a block of code that has since been commented out by @maki49, as you can see above. I decided to keep the original structure intact because I'm unsure if this code will be re-enabled in the future.